Exercicio3.rb

```diff
@@ -0,0 +1,22 @@
+#Exercicio3
+#Mior fator principal
+#Os fatores primos de 13195 são 5, 7, 13 e 29.
+
+#Qual é o maior fator primo do número 600851475143?
+
+def fatorPrimo(n)
```

Comment:
no Ruby por convenção usamos o `snake_case`

aqui deveria ser

```suggestion
def fator_primo(n)
```